chain/client/src/metrics.rs

```diff
@@ -75,3 +75,23 @@ pub static BLOCKS_AHEAD_OF_HEAD: Lazy<IntGauge> = Lazy::new(|| {
     )
     .unwrap()
 });
+pub static SYNC_STATUS: Lazy<IntGauge> =
+    Lazy::new(|| try_create_int_gauge("near_sync_status", "Node sync status").unwrap());
+pub static EPOCH_HEIGHT: Lazy<IntGauge> = Lazy::new(|| {
+    try_create_int_gauge("near_epoch_height", "Height of the epoch at the head of the blockchain")
+        .unwrap()
+});
+pub static PROTOCOL_UPGRADE_BLOCK_HEIGHT: Lazy<IntGauge> = Lazy::new(|| {
+    try_create_int_gauge(
+        "near_protocol_upgrade_block_height",
+        "Estimated block height of the protocol upgrade",
```

Comment:
Just curious, what's the purpose for this metric?

Reply:
It lets validators set alerts about an imminent protocol upgrade. Multiple validators expressed interest in this alert.